core/config/v2/types.go

```diff
@@ -19,6 +20,7 @@ import (
 // Core holds the core configuration. See chainlink.Config for more information.
 type Core struct {
 	// General/misc
+	AuditLogger         *audit.AuditLoggerConfig
```

Comment:
Let's move this down below with the other toml 'table's. There's not really a strict order among them though, so wherever you think makes sense :shrug:

Reply:
I guess I was going for alphabetical order and not mixing any more. I don't feel strongly if you want it moved though

Reply:
Well the tables/structs are below this set of `General/misc` fields since that is how the toml lib encodes it. This one could pair with the `Log` table. :shrug: